package.json

```diff
@@ -50,7 +50,9 @@
     "eslint-plugin-simple-import-sort": "^7.0.0",
     "eslint-plugin-unused-imports": "^2.0.0",
     "husky": "^8.0.1",
-    "jest": "27.5.1",
+    "ignore": "^5.2.0",
+    "jest": "2
```

Comment:
Looks like you added a return character by mistake here?

Reply:
thanks :man_facepalming: